docs: add ES|QL query examples for populating dashboard controls (#7503) (#7505)
elastic/docs-content: explore-analyze/_snippets/esql-query-control-value-examples.md

> When you populate a control with an {{esql}} query, you can shape the values it offers in ways that selecting a field alone doesn't allow. For example, use `WHERE` to limit the values the control offers, instead of surfacing every value in the field. This query offers only the operating systems seen in requests for CSS files from the sample web logs, instead of every operating system in the data:

Author: ```suggestion
When you populate a control with an {{esql}} query, you can shape the values it offers more precisely. 

For example, use `WHERE` to limit the values the control offers, instead of surfacing every value in the field. This query offers only the operating systems seen in requests for CSS files from the sample web logs, instead of every operating system in the data:
```

> When you populate a control with an {{esql}} query, you can shape the values it offers more precisely. 

Author: ```suggestion
With an {{esql}} query, you can shape the values offered in the control more precisely. 
```

> For example, use `WHERE` to limit the values the control offers, instead of surfacing every value in the field. This query offers only the operating systems seen in requests for CSS files from the sample web logs, instead of every operating system in the data:

Author: ```suggestion
For example, use `WHERE` to limit the values the control offers, instead of surfacing every value in the field. The following query offers only the operating systems seen in requests for CSS files from the sample web logs, instead of every operating system in the data:
```
